lib/Subscribers/ImportExport/PersonalDataExporters/NewsletterClicksExporter.php

```diff
@@ -1,9 +1,9 @@
 <?php
-
```

Comment:
Another very common unrelated coding style change in this commit. I don't even think it's our coding style to have that empty line removed.